cns/service/main.go

```diff
@@ -932,7 +933,7 @@ func main() {
 	}
 
 	if !disableTelemetry {
-		go logger.SendHeartBeat(rootCtx, cnsconfig.TelemetrySettings.HeartBeatIntervalInMins)
+		go internal.SendHeartBeat(rootCtx, cnsconfig.TelemetrySettings.HeartBeatIntervalInMins, homeAzMonitor)
```

Comment:
Why don't we check the `cnsConfig.ChannelMode` here as well? If it's not direct, we don't even need to start SendHeartBeat go routine.

Reply:
heartbeat is not specific to Direct channel. we only add azr metric when its Direct channel.